cfme/intelligence/reports/menus.py

```diff
@@ -43,14 +45,13 @@ def is_displayed(self):
         )
 
 
-class ReportMenu(Navigatable):
+@attr.s
+class ReportMenu(BaseEntity):
     """
         This is a fake class mainly needed for navmazing navigation.
 
     """
-    def __init__(self, appliance=None):
-        Navigatable.__init__(self, appliance)
-        self.group = None
+    group = attr.ib(default=None)
```

Comment:
I don't think this should be attr defined, as its not an option that can be passed to the class's constructor, but is set through `go_to_group` method. It should just be a class attribute.